app/views/admin/worldwide_organisations/index.html.erb

```diff
@@ -1,41 +1,50 @@
-<% page_title "Worldwide organisation" %>
+<% content_for :page_title, "Worldwide organisation" %>
+<% content_for :title, "Worldwide organisation" %>
+<% content_for :title_margin_bottom, 6 %>
 
-<h1>Worldwide organisations</h1>
-<%= link_to "Create worldwide organisation", new_admin_worldwide_organisation_path, class: "btn btn-default add-top-margin new_resource", title: "Create a worldwide office" %>
-
-<table class="worldwide_organisation table table-striped table-bordered add-top-margin">
-  <thead>
-    <tr class="table-header">
-      <th width="40%">Name</th>
-      <th width="15%">Translations</th>
-      <th width="20%">Location(s)</th>
-      <th width="15%">Sponsor(s)</th>
-    </tr>
-  </thead>
-  <tbody>
-    <% if @worldwide_organisations.any? %>
-      <% @worldwide_organisations.each do |worldwide_organisation| %>
-        <%= content_tag_for(:tr, worldwide_organisation) do %>
-          <td class="name">
-            <%= link_to worldwide_organisation.name, admin_worldwide_organisation_path(worldwide_organisation), title: "View #{worldwide_organisation.name}" %>
-          </td>
-          <td>
-              <% if worldwide_organisation.non_english_translated_locales.any? %>
-                <%= worldwide_organisation.non_english_translated_locales.map do |locale|
-                  link_to("#{locale.english_language_name} (#{locale.native_language_name})", edit_admin_worldwide_organisation_translation_path(worldwide_organisation, locale.code))
-                end.to_sentence.html_safe %>
-              <% else %>
-                none
-              <% end %>
-          </td>
-          <td><%= worldwide_organisation.world_locations.map {|l| link_to(l.name, [:admin, l.world_location_news])}.to_sentence.html_safe %></td>
-          <td><%= worldwide_organisation.sponsoring_organisations.map {|o| link_to(o.acronym, [:admin, o])}.to_sentence.html_safe %></td>
-        <% end %>
-      <% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
sorry one last thing, generally we've used full width for tables with 3 or more columns. Just wanted to double check this is fine. Probs a question for Nik though.